src/views/ItemView.js

```diff
@@ -597,1138 +484,1342 @@ class ItemView extends React.Component {
-      acc05: !areAllFieldsEmpty(values(itemNotes)),
-      acc06: !areAllFieldsEmpty(values(loanAndAvailability)),
-      acc07: !areAllFieldsEmpty([...values(holdingLocation), ...values(itemLocation)]),
-      acc08: !areAllFieldsEmpty(values(electronicAccess)),
-      acc09: !areAllFieldsEmpty(values(circulationHistory)),
-      acc10: !areAllFieldsEmpty(values(boundWithTitles)),
-      itemAcquisitionAccordion: true,
-    };
-
-    const statisticalCodeContent = !isEmpty(administrativeData.statisticalCodeIds)
-      ? administrativeData.statisticalCodeIds.map(id => ({ codeId: id }))
-      : [{ codeId: '-' }];
-
-    const statisticalCodeFormatter = {
-      'Statistical code type': x => refLookup(referenceTables.statisticalCodeTypes,
-        refLookup(referenceTables.statisticalCodes, get(x, ['codeId'])).statisticalCodeTypeId).name || noValue,
-      'Statistical code name': x => refLookup(referenceTables.statisticalCodes, get(x, ['codeId'])).name || noValue,
-    };
-
-    const electronicAccessFormatter = {
-      'URL relationship': x => refLookup(referenceTables.electronicAccessRelationships,
-        get(x, ['relationshipId'])).name || noValue,
-      'URI': x => {
-        const uri = x?.uri;
-
-        return uri
-          ? (
-            <a
-              href={uri}
-              rel="noreferrer noopener"
-              target="_blank"
-              style={wrappingCell}
-            >
-              {uri}
-            </a>
-          )
-          : noValue;
-      },
-      'Link text': x => get(x, ['linkText']) || noValue,
-      'Materials specified': x => get(x, ['materialsSpecification']) || noValue,
-      'URL public note': x => get(x, ['publicNote']) || noValue,
-    };
+  const emptyNotes = (
+    <Row>
+      <Col xs={1}>
+        <KeyValue
+          label={<FormattedMessage id="ui-inventory.staffOnly" />}
+          value={noValue}
+        />
+      </Col>
+      <Col xs={11}>
+        <KeyValue
+          label={<FormattedMessage id="ui-inventory.note" />}
+          value={noValue}
+        />
+      </Col>
+    </Row>
+  );
 
-    const boundWithTitleFormatter = {
-      'Instance HRID': x => {
+  const layoutNotes = (noteTypes, notes) => {
+    return noteTypes
+      .filter(noteType => notes.find(note => note.itemNoteTypeId === noteType.id))
+      .map((noteType, i) => {
         return (
-          <Link
-            to={`/inventory/view/${x.briefInstance?.id}`}
-            className="instanceHrid"
-          >
-            <span>{x.briefInstance?.hrid}</span>
-          </Link>
+          <Row key={i}>
+            <Col xs={1}>
+              <KeyValue
+                label={<FormattedMessage id="ui-inventory.staffOnly" />}
+                value={get(item, ['notes'], []).map((note, j) => {
+                  if (note.itemNoteTypeId === noteType.id) {
+                    return <div key={j}>{note.staffOnly ? 'Yes' : 'No'}</div>;
+                  }
+
+                  return null;
+                })}
```

Comment:
I would suggest moving this and the ones below to a variable and using `useMemo` so as not to re-render values each time.